test/src/main/java/org/apache/accumulo/test/functional/DeleteRowsIT.java

```diff
@@ -67,7 +67,7 @@ protected int defaultTimeoutSeconds() {
     ROWS.add("{");
   }
 
-  @Test(timeout = 5 * 60 * 1000)
```

Comment:
This may increase the timeout from 5 min to 10 min (the timeout in defaultTimeoutSeconds() for this test)

Reply:
Yeah I went with the higher time to be safe. It looks like 10 minutes is the max time for any IT so I think that is OK.

Reply:
After looking at this test closer and running it a few times... 10 minutes does seem way too high so I reduced the timeout to 5. Slower machines will now be able to use a timeout.factor=2 which would make it 10 anyway.